### PR TITLE
options.targetSelector to take screenshot of just an element

### PR DESCRIPTION
**Pull request by mfolnovic**
Based on https://github.com/dferber90/jsdom-screenshot/pull/4.

Removed `target` and introduced `targetSelector`.

### CODE REVIEW

**Comment by mfolnovic** [comment]
@dferber90 I also removed Enzyme specific documentation that was also added in https://github.com/dferber90/jsdom-screenshot/pull/4, since I'm not using it and wasn't sure how it would work with this new `targetSelector`, hope that's okay.

**Review by dferber90 — APPROVED** [review]

**Comment by dferber90** [comment]
Thanks, I'll take it from here :)

**Comment by mfolnovic** [comment]
@dferber90 sorry for being impatient, are you going to release new version with this change? :angel:

thanks! :blush:

File: yarn.lock [hunk 1]
```diff
@@ -16,43 +16,46 @@
...
     js-tokens "^4.0.0"
 
-acorn-jsx@^4.1.1:
-  version "4.1.1"
-  resolved "https://registry.yarnpkg.com/acorn-jsx/-/acorn-jsx-4.1.1.tgz#e8e41e48ea2fe0c896740610ab6a4ffd8add225e"
-  dependencies:
-    acorn "^5.0.3"
+acorn-jsx@^5.0.0:
+  version "5.0.1"
+  resolved "https://repository.dmz.croz.net/repository/npm-all/acorn-jsx/-/acorn-jsx-5.0.1.tgz#32a064fd925429216a09b141102bfdd185fae40e"
```
Comment (dferber90) [inline yarn.lock:21]: @mfolnovic This doesn't look right. It's not cool to commit these things to the repo, especially as they're really hard to detect.

**Comment by dferber90** [comment]
@mfolnovic Please give maintainers some time. Not everybody is working at your disposal and this is free work. A gentle bump after a few days might be okay, but getting in touch so early is pretty over the top.

I'm currently improving the code you wrote (it was failing the linter and not following prettier), so have some patience. It's only been half an hour.

**Comment by dferber90** [comment]
3.2.0 is out now 👍

**Pull request merged**